templates/layout.jade

```diff
@@ -31,3 +31,11 @@ html(lang='en')
             p &copy; #{ new Date().getFullYear() } #{ locals.owner } &mdash; powered by&nbsp;
               a(href='https://github.com/jnordberg/wintersmith') Wintersmith
               //- please leave the "powered by" if you use the design
+    //- Google analytics
+    script
+     | (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+     | (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+     | m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+     | })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+     | ga('create', 'UA-49880770-1', 'auto');
+     | ga('send', 'pageview');
```

Comment:
Got it, since we are only ever running `build` for production purposes anyways.

Re: GA config and swapping for dev, etc. Eh, I can't really think of any reason why I would want it in dev, unless we started doing custom events or something other than the standard copy/paste the tracking code on the site.